conda/recipes/rmm/meta.yaml

```diff
@@ -38,6 +39,7 @@ requirements:
     - {{ pin_compatible('cudatoolkit', max_pin='x', lower_bound='11.2') }} # cudatoolkit >=11.2,<12.0.0
 {% else %}
     - {{ pin_compatible('cudatoolkit', upper_bound='11.2', lower_bound='11.0') }} # cudatoolkit >=11.0,<11.2
+    - cuda-python {{ cuda_python_version }}
```

Comment:
This pinning should probably be outside the `if cudaMallocAsync == "has_cma"` block, right? If so, move it below the `{% endif %}`.